prometheus/doc/README.md

```diff
@@ -0,0 +1,70 @@
+# Goal
+
+Monitoring all compoments in pai, provide insight on detectiving system/hardware failuring and
+analysing jobs performance.
+
+To accomplish the goal, this module should be:
+* Robust: monitoring should try its best to not crash. Other compoments/people relay on monitoring system to find root causes or trigger alert.
```

Comment:
It's better to only clarify the goal -> (functionality of the monitor part). 

[this module should be] content is service developer's goal and not the service goal and each service need to achieve. So it's better to remove this.